modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -705,6 +748,24 @@ public void testPutDataDefinition() throws Exception {
 		assertValid(getDataDefinition);
 	}
 
+	@Override
+	@Test
+	public DataDefinition
+			testPutSiteDataDefinitionByContentTypeByExternalReferenceCode_addDataDefinition()
+		throws Exception {
+
+		return _addDataDefinition("data-definition-1.json");
```

Comment:
can you apply the last comment here please?

Reply:
Sure